core/src/main/kotlin/notification/service/PushPreferenceService.kt

```diff
@@ -48,7 +49,17 @@ class PushPreferenceServiceImpl(
         pushPreferenceRepository.findByUserId(user.id!!)
             ?.let { PushPreferenceDto(it) }
             ?: PushPreferenceDto(
-                pushPreferences = emptyList(),
+                pushPreferences =
+                    listOf(
+                        PushPreferenceItem(
+                            type = PushPreferenceType.LECTURE_UPDATE,
+                            isEnabled = true,
+                        ),
+                        PushPreferenceItem(
+                            type = PushPreferenceType.VACANCY_NOTIFICATION,
+                            isEnabled = true,
+                        ),
+                    ),
```

Comment:
나중에 타입 추가될 때 확장성을 위해

```suggestion
                pushPreferences = PushPreferenceType.entries.map { PushPreferenceItem(type=it, isEnabled = true) } 
```
이렇게 하면 좋을듯